composer.json

```diff
@@ -41,7 +41,7 @@
         "symfony/monolog-bridge": "~2.6.0",
         "symfony/event-dispatcher": "~2.6.0",
         "pear/pear_exception": "~1.0.0",
-        "matomo/referrer-spam-blacklist": "~1.0",
```

Comment:
That's not the problem. See https://getcomposer.org/doc/articles/versions.md#tilde-version-range-

`~1.0` is the same as '>=1.0 < 2.0.0`

The reason for the older version is because it is (was) pinned in `composer.lock`. This is resolved by d9eef2aa.

Reply:
Okay, I'm not fluent with Composer, my bad. The list has domains which are problematic for me, it just hasn't been released since January, haven't noticed that before.